### PR TITLE
Remediations card optimization related to RHIF-141 & RHIF-142

### PR DESCRIPTION
**Pull request by mhuth**
Implements https://issues.redhat.com/browse/RHIF-142

Depends on the related PR going into the remediations backend: https://github.com/RedHatInsights/insights-remediations/pull/297

### CODE REVIEW

File: src/SmartComponents/Remediations/RunStatus.js [hunk 1]
```diff
@@ -40,56 +38,25 @@ const renderStatusIcon = (status) => ({
...
-                : null
-            }
-            { loaded && !hasData &&
+                :
                 <React.Fragment>
                     <QuestionIcon />
                     <p>{intl.formatMessage(messages.remediationsPlaybookNoActivity)}</p>
                 </React.Fragment>
             }
-            { isCriticalError &&
```
Comment (mkholjuraev) [inline src/SmartComponents/Remediations/RunStatus.js:88]: I think this is an error-handling message and we should keep it.

**Review by mkholjuraev — COMMENTED** [review]
Looks good to me. Just one minor adjustment and then it is good to go.

File: src/AppConstants.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ export const INVENTORY_WARNING_FETCH_URL = `${BASE_URL}${INVENTORY_BASE}/hosts?s
 // Remediations App Constants
 const REMEDIATIONS_BASE = '/remediations/v1';
 export const REMEDIATIONS_FETCH = 'REMEDIATIONS_FETCH';
-export const REMEDIATIONS_FETCH_URL = `${BASE_URL}${REMEDIATIONS_BASE}/remediations?sort=-updated_at&limit=4&offset=0`;
+export const REMEDIATIONS_FETCH_URL = `${BASE_URL}${REMEDIATIONS_BASE}/remediations?sort=-updated_at&limit=4&offset=0&playbook_runs=true`;
 export const REMEDIATIONS_PLAYBOOK_RUNS_FETCH = 'REMEDIATIONS_PLAYBOOK_RUNS_FETCH';
```
Comment (mkholjuraev) [inline src/AppConstants.js:66]: If these constants are not used anymore, we can remove them as well.

File: src/SmartComponents/Remediations/RunStatus.js (hunk 1 above)
Comment (mhuth) [reply src/SmartComponents/Remediations/RunStatus.js:88]: This error only happened when downloading the playbook_runs separately.  Now that the playbook_runs are integrated into REMEDIATIONS_FETCH_URL we don't need this error handling anymore.

File: src/AppConstants.js (hunk 2 above)
Comment (mhuth) [reply src/AppConstants.js]: Done

**Review by mkholjuraev — APPROVED** [review]
LGTM!

**Pull request merged**